### ISSUE TITLE
Latest documentation is missing

### ISSUE THREAD

**Issue by Gcurrivand** [nhn/tui.calendar#1287]
The link to this link works [https://nhn.github.io/tui.calendar/latest/Calendar] but this doc is missing many informations.

For example all the events attached to the calendar cant be find (in this case click event).

![image](https://user-images.githubusercontent.com/114084438/191524676-d1567938-2589-4a95-b3dc-6e622dedea0c.png)

In the previous version everything is fine.

![image](https://user-images.githubusercontent.com/114084438/191524896-ec1ee9ba-78e3-4946-b65c-dfe0765f0ac8.png)

have a good day.

**Comment by adhrinae** [comment]
@Gcurrivand 

Yes, because we rewrote more readable & detailed documentation as Markdown files.

https://github.com/nhn/tui.calendar/blob/main/docs/README.md

Even we left the link inside the docs page. I understand it's not good enough though.

<img width="878" alt="image" src="https://user-images.githubusercontent.com/14539203/191630230-546b2363-42a1-4a31-a00b-9f647848ef6f.png">

But it's not an issue we can handle right now. Closing the issue since it's an intended behavior.
